Sources/SwiftlyCore/SwiftlyCore.swift

```diff
@@ -45,7 +45,7 @@ public protocol InputProvider {
 public var inputProvider: (any InputProvider)?
 
 public func readLine(prompt: String) -> String? {
-    print(prompt, terminator: ": ")
+    print(prompt, terminator: ": \n")
```

Comment:
So does the user input then get output on a new line? So the stdout would look like:
```
Yes or No: 
No   <-- this is my user input
```
?